build/configs/rtl8720e/loadable_apps/defconfig

```diff
@@ -549,29 +534,13 @@ CONFIG_UART2_2STOP=0
 # CONFIG_LPUART6_SERIALDRIVER is not set
 # CONFIG_LPUART7_SERIALDRIVER is not set
 # CONFIG_LPUART8_SERIALDRIVER is not set
-CONFIG_UART3_RXBUFSIZE=1024
-CONFIG_UART3_TXBUFSIZE=1024
-CONFIG_UART3_BAUD=1500000
-CONFIG_UART3_BITS=8
-CONFIG_UART3_PARITY=0
-CONFIG_UART3_2STOP=0
-# CONFIG_UART3_IFLOWCONTROL is not set
-# CONFIG_UART3_OFLOWCONTROL is not set
-CONFIG_UART4_RXBUFSIZE=1024
-CONFIG_UART4_TXBUFSIZE=1024
-CONFIG_UART4_BAUD=1500000
-CONFIG_UART4_BITS=8
-CONFIG_UART4_PARITY=0
-CONFIG_UART4_2STOP=0
-# CONFIG_UART4_IFLOWCONTROL is not set
-# CONFIG_UART4_OFLOWCONTROL is not set
 
 #
 # UART1 Configuration
 #
 
 #
-# UART2 Configuration
+# UART4 Configuration
```

Comment:
I am not sure why these lines have come here.  Either we can move them above to the proper places or we could remove them.

Reply:
I have updated as per the .config
May be its due to the generic Kconfigs. We could modify them in a new PR

Reply:
updated